api/NICE.NextWeb.API/ocelot.production.json

```diff
@@ -16,6 +16,38 @@
         "Region": "OcelotStoryblok"
       }
     },
+    {
+      "DownstreamHostAndPorts": [
+        {
+          "Host": "api.storyblok.com",
+          "Port": 443
+        }
+      ],
+      "DownstreamPathTemplate": "/v1/{everything}",
+      "DownstreamScheme": "https",
+      "UpstreamHttpMethod": [ "Get", "Post" ],
```

Comment:
Discussed removing Post from these new routes (but leaving in legacy route)

Reply:
yep done